lib/modules/auth/login/login_screen.dart

```diff
@@ -2,149 +2,161 @@ import 'package:conditional_builder/conditional_builder.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:fluttertoast/fluttertoast.dart';
-import 'package:shop_app/models/home.dart';
-import 'package:shop_app/models/login_models.dart';
-import 'package:shop_app/modules/auth/login/cubit/login_cubit.dart';
-import 'package:shop_app/modules/auth/login/cubit/login_states.dart';
+import 'package:shop_app/models/login/login_model.dart';
+import 'package:shop_app/modules/auth/login/bloc/login_bloc.dart';
 import 'package:shop_app/modules/auth/register/register_screen.dart';
 import 'package:shop_app/modules/home/home_screen.dart';
+import 'package:shop_app/modules/auth/login/bloc/login_repository.dart';
+
+import 'bloc/login_event.dart';
+import 'bloc/login_state.dart';
 
 class LoginScreen extends StatelessWidget {
   //Form Key
-  GlobalKey<FormState> formKey = GlobalKey<FormState>();
+  final GlobalKey<FormState> formKey = GlobalKey<FormState>();
 
   //TextFormFieldControllers
-  TextEditingController emailController = TextEditingController();
-  TextEditingController passwordController = TextEditingController();
+  final TextEditingController emailController = TextEditingController();
```

Comment:
refactor the Login screen and make sure you follow SOLID standers...
and try to make The UI code using better Way